pkg/guacanalytics/patchPlanning.go

```diff
@@ -190,3 +200,26 @@ func exploreIsDependencyFromDepPkg(ctx context.Context, gqlclient graphql.Client
 
 	return nil
 }
+
+// TODO: impelement this functions
+func exploreIsOccurrence(ctx context.Context, gqlclient graphql.Client, q *queueValues, isOccurrence model.NeighborsNeighborsIsOccurrence) error {
+	// Step 1: Find Artifact attached to package through IsOccurence
+	// -> call .Artifact on isOccurrence
+	// Step 2: Find HasSLSA where Artifact is the builtFrom
```

Comment:
shouldn't this be `exploreHasSLSA` instead?

Reply:
Do you mean I should make a helper called exploreHasSLSA or that the func this is in should be called exploreHasSLSA instead of exploreIsOccurrence?